requirements.txt

```diff
@@ -31,67 +44,80 @@ Flask-Script==2.0.6
 Flask-SQLAlchemy==2.5.1
 Flask-Versioned==0.9.4.post20101221
 future==0.18.2; python_version < '3.0'
-google-auth==1.35.0; python_version < '3.0'
 google-auth==1.23.0; python_version == '3.5'
-google-auth==2.1.0; python_version > '3.5'
-huey==2.3.2
+google-auth==2.6.6; python_version != '3.5'
+huey==2.4.3
 idna==2.10; python_version <= '3.5'
-idna==3.2; python_version > '3.5'
-importlib_metadata==2.1.1; python_version <= '3.5'
-importlib_metadata==4.6.0; python_version > '3.5'
+idna==3.3; python_version > '3.5'
+importlib-metadata==2.1.3; python_version <= '3.5'
+importlib-metadata==4.8.3; python_version == '3.6'
+importlib-metadata==4.11.3; python_version > '3.6'
 importlib-resources==3.3.1; python_version < '3.0'
 importlib-resources==3.2.1; python_version == '3.5'
+importlib-resources==5.4.0; python_version == '3.6'
```

Comment:
Hm, which importlib-resources is used for >3.6?

Reply:
None, it is not used with Python > 3.6